Sample/intertase_firebase_push_notifications_utils.js

```diff
@@ -0,0 +1,49 @@
+import messaging from '@react-native-firebase/messaging';
+
+import {
+	saveDeviceTokenInFreshchat,
+	isFreshchatNotification,
+	handleFreshchatPushNotifications
+} from './freshchat_utils'
+
+export const saveFirebaseDeviceTokenInFreshchat = async () =>{
+	console.log("Saving Device token");
+	await messaging().registerDeviceForRemoteMessages();
+	const token = await messaging().getToken();
+	console.log(token);
```

Comment:
Should we avoid console logging the token?

Reply:
Remove token